ui/partials/perf/graphsctrl.html

```diff
@@ -79,7 +79,16 @@
           <i class="text-warning fa fa-exclamation-circle"></i>
           <a href="perf.html#/alerts?id={{tooltipContent.alertSummary.id}}">
             Alert #{{tooltipContent.alertSummary.id}}</a>
-          <span class="text-muted">({{tooltipContent.alertSummary.getStatusText()}})</span>
+          <span class="text-muted">({{tooltipContent.alertSummary.getStatusText()}}
```

Comment:
Also, to avoid unsightly wrapping issues, could you add a `text-nowrap` class to the span? (see above)